lib/streamlit/elements/map.py

```diff
@@ -97,8 +97,10 @@ def map(
         Parameters
         ----------
         data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table, Iterable, dict, or None
-            The data to be plotted. Must have columns called 'lat', 'lon',
-            'latitude', or 'longitude'.
+            The data to be plotted. Must have two columns called:
+            - latitude called 'lat', 'latitude', 'LAT', 'LATITUDE'
+            - longitude called 'lon', 'longitude', 'LON', 'LONGITUDE'.
```

Comment:
LGTM after addressing the following:

There should be a new line after `called:`

```suggestion
            The data to be plotted. Must have two columns called:
            
            - latitude called 'lat', 'latitude', 'LAT', 'LATITUDE'
            - longitude called 'lon', 'longitude', 'LON', 'LONGITUDE'.
```

else the docs will render like 
![Screenshot 2022-12-02 at 14 40 43](https://user-images.githubusercontent.com/20672874/205258006-d3401bfd-fd21-4adb-8da9-5407583674ae.png)

instead of 
![Screenshot 2022-12-02 at 14 41 42](https://user-images.githubusercontent.com/20672874/205258044-355a3462-4370-4aab-8de1-2614eb421a0f.png)

Reply:
Fixed. I also deleted the repeated word "called"